rpc/src/history.rs

```diff
@@ -192,18 +196,20 @@ impl HistoryNetworkApiServer for HistoryNetworkApi {
         Ok(result)
     }
 
-    /// Get a content from the local database
+    /// Get a content from the local database.
+    ///
+    /// If content is absent, Ok(None) is returned.
```

Comment:
Confused about this comment, if `PossibleHistoryContentValue::ContentAbsent` is returned, this is serialized to `0x` instead of ` None`?

Reply:
Good find. This was a doc comment I made but failed to update as more changes were made.